### ISSUE TITLE
Intergrate into Launcher

### ISSUE THREAD

**Issue by Chen-ygopro** [YGOPro-International/YGOPro-Salvation-Server#2]

**Comment by Chen-ygopro** [comment]
Ranking table needs to be integrated into launcher. This uses trueskill instead of elo.
![image](https://cloud.githubusercontent.com/assets/22610579/20779970/b1879f86-b73d-11e6-9df8-326f3ebd2e47.png)

**Comment by Zayelion** [comment]
This change is for International not Salvation?

**Comment by Chen-ygopro** [comment]
No. I'm working on a js plugin that directly works with the discord api. So all you would have to do is run the command and it pulls the latest ranking into the table. What we have here is just a place holder until it's finished. Automated is the best because it keeps staff from being able to change scores.

**Comment by Chen-ygopro** [comment]
Does salvation need it @Zayelion?

**Comment by Zayelion** [comment]
I agree with your assessment of how it should be done. As long as the end result is pure JS I am ok with the methodology. Salvation selected a point system after approximately one year of debates. We feel it addresses some social issues which are more important than simply knowing "who could be who in a fight", because that concept gets misapplied to other social issues. We concluded it while not as accurate is far less toxic and suite our core objective and values.

If we did true skill we would need a way of having it so it is accessible to the end user only and not public.

There is a trueskill implementation with a DB in the libs/database.js file. it only needs to be required and called.
